src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -5,7 +5,8 @@ export default {
   create(context) {
     return {
       MemberExpression: function(node) {
-        if (node.object.object &&
+        if (!node.computed &&
```

Comment:
Presumably actually determining the value would only work if the value defined in the same file?